packages/exo/test/test-amplify-heap-class-kits.js

```diff
@@ -0,0 +1,115 @@
+// modeled on test-revoke-heap-classes.js
+
+// eslint-disable-next-line import/order
+import { test } from './prepare-test-env-ava.js';
+
+// eslint-disable-next-line import/order
+import { M } from '@endo/patterns';
+import { defineExoClass, defineExoClassKit } from '../src/exo-makers.js';
+
+const UpCounterI = M.interface('UpCounter', {
+  incr: M.call()
+    // TODO M.number() should not be needed to get a better error message
+    .optional(M.and(M.number(), M.gte(0)))
```

Comment:
Yes, and in test-revoke-heap-classes.js. In both cases the TODO comment is irrelevant since these test files do not generate any of the error messages that would be improved.

FWIW, this was blindly copy-pasted from test-heap-classes.js where the TODO comments applies. There (and in test-non-enumerable-methods.js), changing to `M.call().optional(M.gte(0))` works but gives worse error messages.

```
  'In "incr" method of (upCounter): arg 0?: "foo" - Must be >= 0',
```
vs
```
  'In "incr" method of (upCounter): arg 0?: string "foo" - Must be a number'
```

Done, and in test-revoke-heap-classes.js